cbv-context-bizstep.jsonld

```diff
@@ -0,0 +1,47 @@
+{
+    "@context": {
+        "@protected": true,
+        "@version": 1.1,
+        "cbv": "https://ns.gs1.org/cbv/",
+
+        "accepting": "cbv:BizStep-accepting",
```

Comment:
it makes total sense to enumerate the vocabulary elements, you chan check the best practices in schema.org for instance. The @context needs to enumerate the different values because the `@base` would just expand any plain dummy string to `https://ns.gs1.org/cbv/BizStep-dummy` and that is something we want to avoid.

None of the @context defined by standards organizations use @base nor @vocab because of their unintended effects. Normally "by default" solutions have the caveat of being triggered when do you don't want them to be triggered.

Reply:
Answered about schema.org below.

> dummy string to https://ns.gs1.org/cbv/BizStep-dummy

With your context "dummy" is expanded to `<some random base>dummy`: how is that any better?

> None of the `@context` defined by standards organizations use `@base` nor `@vocab` 

That's false: schema.org uses `@vocab`